include/oneapi/dpl/pstl/omp/util.h

```diff
@@ -82,8 +82,9 @@ class __buffer_impl
     ~__buffer_impl() { __allocator_.deallocate(__ptr_, __buf_size_); }
 };
 
-template <typename _ExecutionPolicy, typename _Tp>
-using __buffer = __buffer_impl<::std::decay_t<_ExecutionPolicy>, _Tp>;
+template <class _Tag, typename _ExecutionPolicy, typename _Tp>
+using __buffer = ::std::enable_if_t<::std::is_same_v<_Tag, oneapi::dpl::__internal::__omp_backend_tag>,
```

Comment:
It's unclear why it's done that way. To me it doesn't make sense when buffer is result of `enable_if_t`

Reply:
Fixed